modules/gapi/src/backends/ie/giebackend.cpp

```diff
@@ -243,7 +250,12 @@ struct IEUnit {
             {
                 try
                 {
-                    this_plugin.AddExtension(IE::make_so_pointer<IE::IExtension>(extlib));
+                    auto extension = IE::make_so_pointer<IE::IExtension>(extlib);
+#if INF_ENGINE_RELEASE < 2019020000  // < 2019.R2
+                    this_plugin.AddExtension(extension);
+#else
+                    core.AddExtension(extension, "CPU"); //CPU only??
```

Comment:
I didn't figure out if it's better to leave "CPU" here as in example or there should be something else